src/javase02/task01/Main.java

```diff
@@ -0,0 +1,6 @@
+package javase02.task01;
+
```

Comment:
General comment: use ctrl+alt+L for each file before commit.

Reply:
got it 😎